README.md

```diff
@@ -52,7 +52,12 @@ A `/tmp/serenata-data/irregularities.xz` file will be created. It's a compacted
 Also a target directory (where files are saved) can de passed — for example:
 
 ```console
-$ python rosie.py run chamber_of_deputies /my/serenata/directory/
+$ python rosie.py run chamber_of_deputies --path /my/serenata/directory/
+```
+
+Besides that, if you want to, you can run the project with years filter - for example:
```

Comment:
Line 52 uses a [m-dash](http://www.thepunctuationguide.com/em-dash.html) (instead of an hyphen) before _for example_. This line uses an hyphen. Either we go with m-dash or with hyphen, but swinging between both is weird.

Also a line break between text and mono-spaced block makes the raw markdown way easier to read.